microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -93,31 +111,39 @@ If you need to access more detailed information, use the [getHighEntropyValues()
    :::column-end:::
 :::row-end:::  
 
-## Recommended uses of User-Agent Client Hints  
-
-The set of brands and the associated display order change over time, so you should never hard-code indices into the array of returned brands.  To help ensure you spot similar issues in your website early, the browser includes a `GREASE` brand value that changes over time and is formatted to break because of common string parsing issues.  
+For more information navigate to [getHighEntropyValues()][GithubWicgUaClientHintsGethighentropyvalues].
 
-If you're prevented from using [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection], don't use a hardcoded list of known Chromium-based browsers for verification.  Examples of hardcoded browser names include `Microsoft Edge` and `Google Chrome`.  Reasons why [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] isn't applicable may include the following situation.  
+### Using feature detection with User Agent Client Hints
 
-*   A fix for a Chromium bug in a later version must be avoided and the affected browsers are difficult to detect outside of a verification of the brand and version.  
-    
-Instead, you should verify the `Chromium` brand, which ensures your detection applies to all affected Chromium-based browsers.  
+Combining User-Agent Client Hints with [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] is the best mechanism for browser identification. Microsoft recommends using it whenever possible to:
+* Improve code maintainability.
+* Reduce code fragility.  
+* Reduce code breakage from changes to the User-Agent String.
 
+Use this method to verify the `Chromium` brand and apply detection to all affected Chromium based browsers.
 
 ```javascript
 function isChromium() {
     for (brand_version_pair in navigator.userAgentData.brands) {
-        if (brand_version_pair.brand == "Chromium") {
+        if (brand_version_pair.brand == "Chromium"){
             return true;
         }
     }
     return false;
 }
-```  
+```
+
+Brand names and the associated display order change over time. Avoid hard-coding brand names.
+
+If you cannot use [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection], don't use a hardcoded list of known Chromium-based browsers for verification. Examples of hardcoded browser names include `Microsoft Edge` and `Google Chrome`.  [Feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] may not be available because a fix for a Chromium bug in later versions must be avoided and the affected browsers are difficult to detect.
+
+## User Agent string  

```

Comment:
possibly globally have a hyphen; User-Agent string.  Debatable.  Should be consistent, in any case.  
- Check https://www.bing.com/search?q=%22user%20agent%20string%22
- Check full-text search in repo, it's both ways.  user agent string vs user-agent string

Reply:
Tried to standardize on User-Agent